src/OpenTelemetry/Metrics/MeterProviderBuilderExtensions.cs

```diff
@@ -37,6 +37,54 @@ public static MeterProviderBuilder SetObservationPeriod(this MeterProviderBuilde
             return null;
         }
 
+        /// <summary>
+        /// Sets export period.
+        /// </summary>
+        /// <param name="meterProviderBuilder"><see cref="MeterProviderBuilder"/>.</param>
+        /// <param name="periodMilliseconds">Perion in milliseconds.</param>
+        /// <returns><see cref="MeterProvider"/>.</returns>
+        public static MeterProviderBuilder SetExportPeriod(this MeterProviderBuilder meterProviderBuilder, int periodMilliseconds)
+        {
+            if (meterProviderBuilder is MeterProviderBuilderSdk meterProviderBuilderSdk)
+            {
+                return meterProviderBuilderSdk.SetExportPeriod(periodMilliseconds);
+            }
+
+            return null;
+        }
+
+        /// <summary>
+        /// Add measurement processor.
+        /// </summary>
+        /// <param name="meterProviderBuilder"><see cref="MeterProviderBuilder"/>.</param>
+        /// <param name="processor">Measurement Processors.</param>
+        /// <returns><see cref="MeterProvider"/>.</returns>
+        public static MeterProviderBuilder AddProcessor(this MeterProviderBuilder meterProviderBuilder, MeasurementProcessor processor)
+        {
+            if (meterProviderBuilder is MeterProviderBuilderSdk meterProviderBuilderSdk)
+            {
+                return meterProviderBuilderSdk.AddProcessor(processor);
+            }
+
+            return null;
+        }
+
+        /// <summary>
+        /// Add export processor.
+        /// </summary>
+        /// <param name="meterProviderBuilder"><see cref="MeterProviderBuilder"/>.</param>
+        /// <param name="processor">Measurement Processors.</param>
+        /// <returns><see cref="MeterProvider"/>.</returns>
+        public static MeterProviderBuilder AddExportProcessor(this MeterProviderBuilder meterProviderBuilder, ExportMetricProcessor processor)
```

Comment:
we don't need this now. Lets just keep AddProcessor alone.

Reply:
We need a way to distinguish between Pre-Aggregation processors vs Post-Aggregation processors. These processors take different data.